deployment/updates/scripts/xml/notifications/entryClipJobCompletedHttpNotificationTemplate.xml

```diff
@@ -0,0 +1,96 @@
+                            <code>!is_null(entryPeer::retrieveByPk($scope->getObject()->getId())) ? entryPeer::retrieveByPk($scope->getObject()->getId())->getFlowType(): false</code>
+                        </field>
+                        <values objectType="array">
+                            <item objectType="KalturaStringValue">
+                                <value>2</value> <!-- EntryFlowType::TRIM_CONCAT -->
+                            </item>
+                        </values>
+                    </item>
+                </eventConditions>
+                <contentParameters objectType="array">
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>entry_id</key>
+                        <description>Entry id of entry which trimmed</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>!is_null(entryPeer::retrieveByPk($scope->getObject()->getId())) ? $scope->getObject()->getReplacedEntryId() : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>clip_trim_offset</key>
+                        <description>Clip trim offset</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>!is_null($entry = entryPeer::retrieveByPk($scope->getObject()->getId())) &amp;&amp; !is_null($operationAttributes = $entry->getOperationAttributes()) &amp;&amp; $operationAttributes[0] instanceof kOperationAttributes ? $operationAttributes[0]->getOffset() : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>clip_trim_duration</key>
+                        <description>Clip trim duration</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>!is_null($entry = entryPeer::retrieveByPk($scope->getObject()->getId())) &amp;&amp; !is_null($operationAttributes = $entry->getOperationAttributes()) &amp;&amp; $operationAttributes[0] instanceof kOperationAttributes ? $operationAttributes[0]->getDuration() : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
+                        <key>reference_id</key>
+                        <description>Reference Id of entry</description>
+                        <value objectType="KalturaEvalStringField">
+                            <code>!is_null(entryPeer::retrieveByPk($scope->getObject()->getReplacedEntryId())) ? entryPeer::retrieveByPk($scope->getObject()->getReplacedEntryId())->getReferenceId() : ''</code>
+                        </value>
+                    </item>
+                    <item objectType="KalturaEventNotificationParameter">
```

Comment:
@TalDubovKaltura not sure what this parameter is? time() returns the current timestamp. This is the time that the event notification job was started, but nothing to do with the trimming flow. If the point is to provide the timestamp of when the trimming process was complete then
a. best use the temp entry's updated_at (which is set to the moment the temp entry was deleted)
b. please change the description to be more specific.

Reply:
I change the description, this time already used in the start trim event notification presenting the time the event was triggered